src/ert/parsing/lark_parser.py

```diff
@@ -231,17 +240,52 @@ def _tree_to_dict(
     return config_dict
 
 
+ArgPairList = List[Tuple[FileContextToken]]
+ParsedArgList = List[Union[FileContextToken, ArgPairList]]
+
+
 def _substitute_args(
-    args: List[Any], constraints: SchemaItem, defines: Defines
-) -> List[Any]:
+    args: ParsedArgList,
+    constraints: SchemaItem,
+    defines: Defines,
+) -> ParsedArgList:
     if constraints.substitute_from < 1:
         return args
-    return [
-        _substitute(defines, x, constraints.expand_envvar)
-        if i + 1 >= constraints.substitute_from and isinstance(x, FileContextToken)
-        else x
-        for i, x in enumerate(args)
-    ]
+
+    def substitute_arglist_tuple(
+        tup: Tuple[FileContextToken],
+    ) -> Tuple[FileContextToken]:
+        tuple_kw = tup[0]
+        tuple_args = tup[1:]
```

Comment:
Fixed. 

Was not really sure if it can somehow in some cases receive more, though it seems it is not supported in currently:

```
FORWARD_MODEL MAKE_SYMLINK(<U>=<A> <B> <C>)
FORWARD_MODEL MAKE_SYMLINK(<U>=<A>,<B>,<C>)
```

fails in the new parsre, so assuming it is only two